qutip/qip/operations/gates.py

```diff
@@ -106,6 +107,178 @@ def rz(phi, N=None, target=0):
                      [0, np.exp(1j * phi / 2)]])
 
 
+def x_gate(N=None, target=0):
+    """Single-qubit rotation through pi radians around the x-axis.
+
+    Returns
+    -------
+    result : qobj
```

Comment:
in general "qobj"---> "Qobj" in docstring. Even better would be ":class: `qutip.Qobj`" as that should get picked up by Sphinx in the docs and make it clieckable.